includes/Handler/class-mf2.php

```diff
@@ -595,6 +595,19 @@ protected function urls_match( $url1, $url2 ) {
 	protected function get_class_mapper() {
 		$class_mapper = array();
 
+		/*
+		 * These classes represent the source sending a webmention based on its display of a mention sent by the target.
+		 * This sets the type to update, indicating that the purpose of the webmention is to notify of an update to the original webmention.
+		 */
+		$class_mapper['comment']  = 'update';
+		$class_mapper['like']     = 'update';
+		$class_mapper['favorite'] = 'update';
+		$class_mapper['repost']   = 'update';
+		$class_mapper['bookmark'] = 'update';
+		$class_mapper['read']     = 'read';
```

Comment:
Shouldn't this and the following two be also "update"?

Reply:
We should also maybe prefix this with something, because there might be an `Update` type in the future?!?

Reply:
Yes, the read is just my mistake not updating it when I moved it out of place. No issue with a prefix. How about source_update?

Reply:
🤔

So this case should only happen if someone has a post with reactions and updates that post?!?

Reply:
Correct.

Reply:
What about something like `error` or `failure`, then we would not need to prefix it. `Update` sound like that it is an expected ping!?!

Reply:
Let's say that my Post A is a reply to Post B. Post B marks it up as a response. then Post B's author updates the post and sends a webmention to Post A. If Post A has a reply-context on it, then the webmention sent by Post B should update that reply context. We don't have that feature, and it wouldn't be webmentions doing it necessarily, but by not having the hook that acknowledges it...

It isn't an error so much as an unhandled feature right now.

Reply:
You mean something like a reverse update? Kind of "you referenced my blog and I updated it, maybe you are interested in that change" like?

Reply:
It's not a bug it's a (not yet defined/implemented) feature? ☺️

Reply:
Exactly. So...the solution is to have the action that someone could add the feature to, but otherwise for now discard the mention.

Reply:
Makes also sense! Sorry for standing on the hose for so long!

But I think this would be a target_update then?!?